DQM/Integration/python/clients/scal_dqm_sourceclient-live_cfg.py

```diff
@@ -76,7 +76,7 @@
 process.dqmmodules = cms.Sequence(process.dqmEnv + process.dqmSaver )#+ process.dqmSaverPB)
 process.evfDQMmodulesPath = cms.Path(
                               process.l1GtUnpack*
```

Comment:
Hi, just a couple of quick comments:
   * from the exception message [1] log [here](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-1f6165/43146/unitTests/failed.html) , actually (also) `l1GtUnpack` needs to be changed.  
   * the SCAL FED has been removed from DAQ since the end of Run 2. Do we still need to support this client? The fact that  [CMSALCAFAST-94](https://its.cern.ch/jira/browse/CMSALCAFAST-94) succeeded without these changes is proof that this client is not run anymore in live data-taking.
   
   
[1]
```
----- Begin Fatal Exception 29-Nov-2024 00:46:25 CET-----------------------
An exception of category 'NoRecord' occurred while
   [0] Processing  Event run: 355380 lumi: 19 event: 26716348 stream: 0
   [1] Running path 'evfDQMmodulesPath'
   [2] Calling method for module L1GlobalTriggerRawToDigi/'l1GtUnpack'
Exception Message:
No "L1MuTriggerScalesRcd" record found in the EventSetup.

 Please add an ESSource or ESProducer that delivers such a record.
----- End Fatal Exception -------------------------------------------------
```

Reply:
Thank you @mmusich 
Indeed, removing that test at all would be a much simpler solution: if you also support it, I'll do so. (Now I am traveling, I will do it as soon as I will be able to)

Reply:
Yeah, that would make sense. But I'd let it be done in the proper way by @cms-sw/l1-l2 (or by whoever else wants to do it).
For the sake of just having the updated GTs running without crashes in the CMSSW unit tests system let just comment out here the obsolete one.

Reply:
@cms-sw/dqm-l2 can you comment on a possible removal of the client from the release already in this PR?

Reply:
no objection to removing the client outside of pp data taking